.travis.yml

```diff
@@ -15,3 +15,4 @@ script:
  - pipenv run coverage run --branch --source pyramid_heroku -m pytest pyramid_heroku
  - pipenv run coverage html
  - pipenv run coverage report --fail-under=100
+ - pipenv run black pyramid_heroku/
```

Comment:
You need `--check` that will make the build failed if something changed.

Reply:
Sry about that, done.